vividus-tests/src/main/resources/story/integration/ActionSteps.story

```diff
@@ -3,10 +3,73 @@ Description: Integration tests for ActionSteps class.
 Meta:
     @epic vividus-plugin-web-app
 
+Scenario: Action verification MOVE_BY_OFFSET
+Given I am on a page with the URL '${vividus-test-site-url}/mouseEvents.html'
+Then number of elements found by `By.xpath(//*[@id='moveCount' and text()='0'])` is equal to `1`
+When I execute sequence of actions:
+|type          |argument                                   |
+|MOVE_TO       |By.xpath(//div[contains(., 'Mouse move!')])|
+|MOVE_BY_OFFSET|(0, 10)                                    |
+|MOVE_BY_OFFSET|(0, -10)                                   |
+|MOVE_BY_OFFSET|(0, 10)                                    |
+|MOVE_BY_OFFSET|(0, -10)                                   |
+Then number of elements found by `By.xpath(//*[@id='moveCount' and text()='5'])` is equal to `1`
+
 Scenario: Action verification MOVE_TO
-Given I am on a page with the URL 'https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onmousemove_over_enter'
-When I switch to a frame with the attribute 'id'='iframeResult'
+Given I am on a page with the URL '${vividus-test-site-url}/mouseEvents.html'
```

Comment:
Why not refresh?

Reply:
to make tests independent of their position + consistency